src/normalizer.ts

```diff
@@ -298,7 +307,30 @@ export const normalizeText = (t: TextChild): InternalTextPart[] => {
   }
 };
 
+const normalizeTextType = (
+  node: React.ReactElement,
+  normalizedCoordinates: Record<string, `${number}%` | number>
+) => {
+  const style = node.props.style;
+  return {
+    text:
+      node.props.children !== undefined
+        ? normalizeText(node.props.children)
+        : [],
+    style: {
+      ...style,
+      ...normalizedCoordinates,
+      color: style.color ? normalizeHexColor(style.color) : null,
+      fontFace: style.fontFace ?? DEFAULT_FONT_FACE,
+      fontSize: style.fontSize ?? DEFAULT_FONT_SIZE,
+    },
+    colSpan: node.props.colSpan,
+    rowSpan: node.props.rowSpan,
```

Comment:
Should these be here since this is a generic function for text nodes?

Reply:
Done ✅